x-pack/plugins/enterprise_search/server/routes/enterprise_search/connectors.ts

```diff
@@ -137,4 +140,57 @@ export function registerConnectorRoutes({ router, log }: RouteDependencies) {
       return response.ok({ body: result });
     })
   );
+
+  router.post(
```

Comment:
I think these should be puts? Since they're updating? I don't really care that much tho 🤷